zk-sdk-wasm-js/src/encryption/elgamal.rs

```diff
@@ -1,36 +1,34 @@
 use {
-    crate::encryption::pedersen::{WasmPedersenCommitment, WasmPedersenOpening},
+    crate::encryption::pedersen::{PedersenCommitment, PedersenOpening},
     js_sys::Uint8Array,
     solana_zk_sdk::encryption::{
-        elgamal::{
-            DecryptHandle, ElGamalCiphertext, ElGamalKeypair, ElGamalPubkey, ElGamalSecretKey,
-        },
-        DECRYPT_HANDLE_LEN, ELGAMAL_CIPHERTEXT_LEN, ELGAMAL_PUBKEY_LEN, ELGAMAL_SECRET_KEY_LEN,
+        elgamal, DECRYPT_HANDLE_LEN, ELGAMAL_CIPHERTEXT_LEN, ELGAMAL_PUBKEY_LEN,
+        ELGAMAL_SECRET_KEY_LEN,
     },
     wasm_bindgen::prelude::{wasm_bindgen, JsValue},
 };
 
-#[wasm_bindgen(js_name = "ElGamalPubkey")]
-pub struct WasmElGamalPubkey {
-    pub(crate) inner: ElGamalPubkey,
+#[wasm_bindgen]
+pub struct ElGamalPubkey {
+    pub(crate) inner: elgamal::ElGamalPubkey,
 }
 
-crate::conversion::impl_inner_conversion!(WasmElGamalPubkey, ElGamalPubkey);
+crate::conversion::impl_inner_conversion!(ElGamalPubkey, elgamal::ElGamalPubkey);
 
 #[wasm_bindgen]
-impl WasmElGamalPubkey {
+impl ElGamalPubkey {
     /// Creates an ElGamal public key from a secret key.
     #[wasm_bindgen(js_name = "fromSecretKey")]
-    pub fn from_secret_key(secret_key: &WasmElGamalSecretKey) -> Self {
+    pub fn from_secret_key(secret_key: &ElGamalSecretKey) -> Self {
         Self {
-            inner: ElGamalPubkey::new(secret_key),
+            inner: elgamal::ElGamalPubkey::new(secret_key),
         }
     }
 
     /// Deserializes an ElGamal public key from a byte slice.
     /// Throws an error if the bytes are invalid.
     #[wasm_bindgen(js_name = "fromBytes")]
-    pub fn from_bytes(uint8_array: Uint8Array) -> Result<WasmElGamalPubkey, JsValue> {
+    pub fn from_bytes(uint8_array: Uint8Array) -> Result<ElGamalPubkey, JsValue> {
```

Comment:
no need to do anything, but I found these kinds of refactors were just a little bit easier when using `Self` as much as possible inside of impl blocks
```suggestion
    pub fn from_bytes(uint8_array: Uint8Array) -> Result<Self, JsValue> {
```